views/js/qtiCreator/widgets/interactions/graphicOrderInteraction/states/Question.js

```diff
@@ -173,21 +183,21 @@ define([
     /**
      * Exit the question state, leave the room cleaned up
      */
-    var exitQuestionState = function initQuestionState(){
-        var widget      = this.widget;
+    var exitQuestionState = function exitQuestionState() {
```

Comment:
**chore:** It would be better to only declare the function instead of a function expression. I would avoid the need of fixing the name... Unless the reference needs to be changed later of course, but most of the time functions are static.

**suggestion:**
```suggestion
    function exitQuestionState() {
```